apps/full-site-editing/package.json

```diff
@@ -81,6 +81,7 @@
 		"classnames": "2.2.6",
 		"lodash": "*",
 		"moment": "*",
-		"newspack-blocks": "github:Automattic/newspack-blocks#v1.2.1"
+		"newspack-blocks": "github:Automattic/newspack-blocks#v1.2.1",
```

Comment:
does the newspack version need updated? (not sure if this is just a hotfix :))

Reply:
No idea, but I updated it nonetheless :)